grpc/go-cubic/cubicpb/gw/cubic.pb.gw.go-E

```diff
@@ -0,0 +1,299 @@
+// Code generated by protoc-gen-grpc-gateway. DO NOT EDIT.
```

Comment:
What is this `.go-E` file for?

Reply:
I don't know where that came from.  :/  Removed.